code/modules/hydroponics/grown/nettle.dm

```diff
@@ -11,7 +11,7 @@
 	growthstages = 5
 	genes = list(/datum/plant_gene/trait/repeated_harvest, /datum/plant_gene/trait/plant_type/weed_hardy)
 	mutatelist = list(/obj/item/seeds/nettle/death)
-	reagents_add = list("sacid" = 0.5)
+	reagents_add = list("sacid" = 0.1)
```

Comment:
Jughu, gylcerol requires 3 corn oil and 1 sacid, to produce 1 gylcerol. at 50% sacid, putting that gene in corn leaves a lot of leftover acid. upping corn oil and reducing sacaid makes it so that the reaction ONLY produces gylcerol if put in the same plant. at 100 potency, it'd be 30 units of corn oil and 10 units of sulphuric acid, which would produce 10 units of gylcerol. This is not nerfing nettles, on the contrary, its actually buffing them.